cmd/integration/commands/reset_state.go

```diff
@@ -131,11 +131,7 @@ func printStages(tx kv.Tx, snapshots *freezeblocks.RoSnapshots, borSn *freezeblo
 	if err != nil {
```

Comment:
maybe rename s1 to ethSeq or something more clear? we don't need s1 naming due to removing s2 I suppose

Reply:
done